src/analyses/invariant_set_domain.cpp

```diff
@@ -23,6 +23,7 @@ void invariant_set_domaint::transform(
   {
   case GOTO:
     {
+      // GOTO implies from and to are comparable
```

Comment:
Comment is at a slightly weird place; also: another opportunity to introduce `std::next`.